mlir/lib/Dialect/MIOpen/Tuning/ConvContext.cpp

```diff
@@ -36,10 +36,9 @@ void populateSeqVal(const ArrayAttr &seqAttr,
     }
   }
 }
-} // namespace
 
-ConvolutionContext populateConvContext(Operation *op) {
-  miopen::ConvOpType opType = miopen::obtainConvDirection(op);
+ConvolutionContext mlir::miopen::populateConvContext(Operation *op) {
+  ConvOpType opType = obtainConvDirection(op);
```

Comment:
Why do you need to add the namespace around this (with added using statements)

Reply:
I was getting linker errors if I didn't